pkgs/development/libraries/pipewire/default.nix

```diff
@@ -133,6 +137,7 @@ let
       "-Dbluez5-backend-hsphfpd=${mesonEnable hsphfpdSupport}"
       "-Dsysconfdir=/etc"
       "-Dpipewire_confdata_dir=${placeholder "lib"}/share/pipewire"
+      "-Dsession-managers=${mesonList (lib.optional withMediaSession "media-session")}"
```

Comment:
Is this really required? We are explicitly patching out the part from the config template where it substitutes the session manager binary.

Reply:
Ah right, this chooses which ones are built. `default-session-manager` is what gets written to the config. In that case it would be nice if we could stop building examples and just build the media session with this flag. Looks like that is not possible at the moment though.

Reply:
Yeah I'm not sure if/why examples is still necessary? just `session-managers=[x]` should be good enough.

Reply:
Upstream ~~being lazy~~ having other priorities. The media session code is under `src/examples/`and the build files don't even touch that directory if examples are disabled.

Reply:
The [meson manual](https://mesonbuild.com/Build-options.html#using-build-options) describes the syntax.